examples/simple_host.rs

```diff
@@ -15,10 +16,11 @@ impl Host for SampleHost {
     }
 }
 
+#[allow(dead_code)]
 fn main() {
     // This is an example of a plugin being loaded. Change this to the appropriate path.
     let path = Path::new(
-        "/Library/Audio/Plug-Ins/VST/beyerdynamicVS.vst/Contents/MacOS/beyerdynamicVS"
+        "/Library/Audio/Plug-Ins/VST/beyerdynamicVS.vst/Contents/MacOS/beyerdynamicVS",
```

Comment:
Let's get rid of this weirdly specific and potentially confusing path and replace it with something easy like `"/path/to/my/plugin"` - or can we do something in the cargo.toml to set this like an env variable instead of putting it in code?  Is there a best practice for that?

Reply:
the path should be passed as a command line argument

Reply:
same for this. do it in a separate request

Reply:
agree with @zyvitski it should be passed as a command line arg

Reply:
Cool.  I'll open up an issue for doing this later.